README.md

```diff
@@ -42,8 +42,6 @@ There are various settings available for the extension now. All the Apama config
   * Langserver.Port is the port the Langserver is running on.
   * Langserver.MaxErrors is the maximum number of diagnostics that should be returned by the LSP (INACTIVE)
 
-There are also some placeholders for using EPL with [Cumulocity IoT](https://www.softwareag.cloud/site/product/cumulocity-iot.html). these will become active in a later release (>v1.0).
```

Comment:
Any reason we're not just replacing this with https://www.cumulocity.com   ?

Reply:
Because I'm entirely uncertain whether any of this Cumulocity integration works, and I don't feel like documenting stuff that doesn't work.